config/environments/test.rb

```diff
@@ -46,7 +46,7 @@
   config.action_mailer.delivery_method = :test
   # This fills in the 'from' field which is required to run feature
   # tests on circleci
-  config.action_mailer.default_options = { from: "localhost" }
+  # config.action_mailer.default_options = { from: "localhost" }
```

Comment:
What's the reason for this change?

Reply:
Presumably we will need it once feature tests are turned back on (which is a ticket in our next sprint). If this is not causing an issue currently can we retain this line. You can remove the reference to CircleCI